doc/modules/mod_last.md

```diff
@@ -6,16 +6,33 @@ Use with caution, as it was observed that a user disconnect spike might result i
 
 ### Options
 
-* **iqdisc** (default: `one_queue`)
-* **backend** (atom, default: `mnesia`): Storage backend. Currently `mnesia`, `rdbms` and `riak` are supported.
+#### `modules.mod_last.iqdisc`
+* **Syntax:** string
```

Comment:
I think strings lack `""` in this module

Reply:
yes they lacked in apparently most of my modules :( now it should be fixed